up42/tasking.py

```diff
@@ -80,31 +82,21 @@ def construct_order_parameters(
                 )
             ```
         """
-        order_parameters: order.OrderParams = {
-            "dataProduct": data_product_id,
-            "params": {
-                "displayName": name,
-                "acquisitionStart": utils.format_time(acquisition_start),
-                "acquisitionEnd": utils.format_time(acquisition_end, set_end_of_day=True),
-            },
-        }
+        schema = self.get_data_product_schema(data_product_id)
+        params: dict[str, Any] = {param: None for param in schema["required"]}
+        params["displayName"] = name
+        params["acquisitionStart"] = utils.format_time(acquisition_start)
+        params["acquisitionEnd"] = utils.format_time(acquisition_end, set_end_of_day=True)
+        order_parameters: order.OrderParams = {"dataProduct": data_product_id, "params": params}
```

Comment:
do we need typing hint here?

Reply:
If we don't add type, wouldn't be a dict type.